### ISSUE TITLE
BUG - Sync Jellyfin

### ISSUE THREAD

**Issue by VampiricAlien** [CollinHeist/TitleCardMaker#395]
### Installation

Docker - master tag

### Describe the Bug

Was trying to set this up in docker but when I run python main.py --sync I get an error. I was trying to create the sync files.

verify ssl doesn't change the outcome.

### Steps to reproduce the behavior

1. 
2.
...

### Preference File

```yaml
jellyfin:
  url: http://ipaddress:8096/
  api_key: API key
  username: User
  verify_ssl: true
  watched_style: unique
  unwatched_style: blur
  filesize_limit: 4 MB
  sync:
  - file: ./yaml/jellyfin_sync.yml
    mode: append
    compact_mode: true
    add_template: my_template
    libraries:
    - TV Shows
    exclusions:
    - series:
    - yaml: ./yaml/ignore.yml
```

### Debug Log

Traceback (most recent call last):
  File "/maker/main.py", line 255, in <module>
    Manager(check_tautulli=False).sync_series_files()
  File "/maker/modules/Manager.py", line 59, in __init__
    self.jellyfin_interface = JellyfinInterface(
  File "/maker/modules/JellyfinInterface.py", line 89, in __init__
    self.libraries = self._map_libraries()
  File "/maker/modules/JellyfinInterface.py", line 136, in _map_libraries
    return {
  File "/maker/modules/JellyfinInterface.py", line 139, in <dictcomp>
    if library['CollectionType'] == 'tvshows'
KeyError: 'CollectionType'

**Comment by CollinHeist** [comment]
@VampiricAlien Hey - you must be on an old version of TCM, that is a "bug" caused by non-TV/Movie libraries that was fixed in v1.14.1. Can you update and let me know if the problem goes away?

**Comment by CollinHeist** [comment]
@VampiricAlien Your exclusion YAML file `./yaml/ignore.yml` must be invalid/empty, You'll want to either add a blank set in `series` or just remove that completely
